11_webui.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+set -eux
+source common.sh
+
+figlet "Deploying web ui" | lolcat
+eval "$(go env)"
+
+
+export UIPATH="$GOPATH/src/github.com/kubevirt/web-ui-operator"
+
+oc new-project kubevirt-web-ui
+cd $UIPATH/web-ui-operator/deploy
+oc apply -f service_account.yaml
+oc adm policy add-scc-to-user anyuid -z kubevirt-web-ui-operator
+oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:kubevirt-web-ui:kubevirt-web-ui-operator
+oc apply -f role.yaml
+oc apply -f role_binding.yaml
+oc apply -f role_binding_kube-public.yaml
+oc apply -f crds/kubevirt_v1alpha1_kwebui_crd.yaml
+oc apply -f operator.yaml
+#echo -e "  openshift_master_default_subdomain: apps.ostest.test.metalkube.org\n  public_master_hostname: api.ostest.test.metalkube.org:6443" >> deploy/crds/kubevirt_v1alpha1_kwebui_cr.yaml
+oc apply -f crds/kubevirt_v1alpha1_kwebui_cr.yaml
```

Comment:
Is `openshift-console` (already) installed as well?

If so, please add role/role-bidings for it as well (see operator's README)
If not, this CR needs to be updated for additional parameters (check README)

Reply:
openshift console is already installed indeed, role bindings were added

Reply:
This file can be updated to choose branding from "okdvirt" or "openshiftvirt"

Reply:
Thanks to https://github.com/kubevirt/web-ui-operator/pull/37 , the installed web-ui tag is `:latest`.
I guess it's as expected, but just highlighting the fact.